app/Http/Client/Alchemy/AlchemyPendingRequest.php

```diff
@@ -707,6 +708,16 @@ private function getNftV2ApiUrl(): string
         return 'https://'.self::$apiUrlPlaceholder.'.g.alchemy.com/nft/v2/';
     }
 
+    private function getTokenType(string $type): TokenType
```

Comment:
Shouldn't this be in the `app/Enums/TokenType.php`? So you could use it in `app/Http/Client/Moralis/MoralisPendingRequest.php`

Reply:
I agree with that point.